pkg/rpc/server/server.go

```diff
@@ -1197,7 +1206,29 @@ func (s *Server) runScriptInVM(t trigger.Type, script []byte, tx *transaction.Tr
 
 	vm := s.chain.GetTestVM(t, tx, b)
 	vm.GasLimit = int64(s.config.MaxGasInvoke)
-	vm.LoadScriptWithFlags(script, callflag.All)
+	if t == trigger.Verification {
```

Comment:
I really don't like this code in the RPC server. Maybe we can reuse some of the core/contract functionality for it?

Reply:
Fixed by exposing `initVerificationVM`. Are we OK with that?